portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -41,6 +41,19 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.getWriter().println(json);
   }
 
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    // Get the input from the form.
+    String commentText = request.getParameter("comment-text");
+    
+    // Add comment to list
```

Comment:
Once you rename `comments` to `commentList`, you won't need this comment since the code would be self-explanatory.